linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/force_opensc_card_drivers/bash/shared.sh

```diff
@@ -7,8 +7,8 @@
 . /usr/share/scap-security-guide/remediation_functions
 populate var_smartcard_drivers
 
-grep -qs "force_card_driver =" /etc/opensc*.conf && \
-	sed -i "s/force_card_driver =.*/force_card_driver = $var_smartcard_drivers;/g" /etc/opensc*.conf
-if ! [ $? -eq 0 ]; then
-	sed -i "s/.*force_card_driver =.*/        force_card_driver = $var_smartcard_drivers;/g" /etc/opensc*.conf
+OPENSC_TOOL="/usr/bin/opensc-tool"
+
+if [ -f "${OPENSC_TOOL}" ]; then
+    ${OPENSC_TOOL} -S app:default:force_card_driver:$var_smartcard_drivers
```

Comment:
This configuration option does not have any effect in RHEL8 so I do not think it makes sense to bother with it. By specifying the `card_drivers` to `cac`, setting the `force_card_driver` to the same value does not have any effect in RHEL7 either so I propose to drop this step completely (probably as part of different housekeeping PR).